app/lib/frontend/templates/views/pkg/liked_package_list.mustache

```diff
@@ -0,0 +1,25 @@
+{{! Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+    for details. All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file. }}
+
+<ul class="package-list">
+  {{#packages}}
+    <li class="list-item -compact">
+      <button
+        class="mdc-button mdc-button--unelevated -pub-like-button"
+        data-mdc-auto-init="MDCRipple"
+        data-package="{{name}}"
+        data-thumb_up_outlined="{{thumb_up_outlined}}"
+        data-thumb_up_filled="{{thumb_up_filled}}">
```

Comment:
Why not just use `{{{static_assets.img__thumb-up-filled-24px_svg}}}`

Reply:
Done